docs/docsite/rst/scenario_guides/kubernetes_scenarios/scenario_k8s_object.rst

```diff
@@ -0,0 +1,89 @@
+.. _k8s_object_template:
+
+*******************
+Creating K8S object
+*******************
+
+.. contents:: Topics
+
+Introduction
+============
+
+This guide will show you how to utilize Ansible to create Kuberentes objects such as Pods, Deployments, and Secrets.
+
+Scenario Requirements
+=====================
+
+* Software
+
+    * Ansible 2.9.10 or later must be installed
+
+    * The Python modules ``openshift`` and ``kubernetes`` must be installed on the Ansible controller (or Target host if not executing against localhost)
+
+    * Kubernetes Cluster
+
+    * Kubectl binary installed on the Ansible controller
+
+
+* Access / Credentials
+
+    * Kubeconfig configured with the given Kubernetes cluster
+
+
+Assumptions
+===========
+
+- User has required level of authorization to create, delete and update resources on the given Kuberentes cluster.
+
+Caveats
+=======
+
+- community.kubernetes 1.1.0 is going to migrate to `kubernetes.core <https://github.com/ansible-collections/kubernetes.core>`_
+
+Example Description
+===================
+
+In this use case / example, we will be creatin a Pod in the given Kubernetes Cluster.  The following Ansible playbook showcases the basic parameters that are needed for this.
+
+.. literalinclude:: task_outputs/Add_a_pod_using_k8s.yaml
+
+Since Ansible utilizes the Kubernetes API to perform actions, in this use case we will be connecting directly to the Kubernetes cluster.
+
+To begin, there are a few bits of information we will need. Here you are using Kubeconfig which is pre-configured in your machine. The Kubeconfig is generally located at ``~/.kube/config``. It highly recommended to store sensitive information such as password, user certificates in a more secure fashion using :ref:`ansible-vault` or using `Ansible Tower credentials <https://docs.ansible.com/ansible-tower/latest/html/userguide/credentials.html>`_.
+
+Now you need to supply the information about the Pod which will be created. Using ``definition`` parameter of ``community.kubernetes.k8s`` module, you are specifying `PodTemplate <https://kubernetes.io/docs/concepts/workloads/pods/#pod-templates>`_. This PodTemplate is identical to what you provide to ``kubectl`` command.
+
+What to expect
+--------------
+
+- You will see a bit of JSON output after this playbook completes. This output shows various parameters that are returned from the module and from cluster about the newly created Pod.
+
+.. literalinclude:: task_outputs/Add_a_pod_using_k8s.json
+
+- In above example, 'changed' is ``True`` which notifies that the Pod creation started on the given cluster. This can take some time depending on your environment.
```

Comment:
```suggestion
- In the above example, 'changed' is ``True`` which notifies that the Pod creation started on the given cluster. This can take some time depending on your environment.
```